components/CardList/CardList.styled.js

```diff
@@ -24,3 +24,21 @@ export const Input = styled.input`
 export const Results = styled.p`
   text-align: center;
 `;
+export const FilterContainer = styled.section`
+  margin: 1em;
+  display: flex;
+`;
+
+export const FilterSelect = styled.select`
+  padding: 0.5rem;
+  border: none;
+  border-radius: 5px;
+  box-shadow: 0 2px 8px rgba(0, 0, 0, 0.26);
+  transition: width 0.3s ease-in-out;
+  width: 30%;
+  margin: 0 auto;
+
+  &:focus {
+    outline: none;
```

Comment:
For accessibility reasons we do not want to remove the outline for interactive elements. [See MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/outline#accessibility_concerns)

Reply:
thank you for your feedback , i have already remove it.